naming-scheme.rst

```diff
@@ -55,8 +57,18 @@ To rename the binary RPM package to ``python2-<srcname>``, you should build it a
 The change should look like this:
 
 .. literalinclude:: specs/module.spec
-	:diff: specs/module.spec.orig
-	:lines: 15-19,21-32,43
+   :diff: specs/module.spec.orig
+   :lines: 15-19,21-32,43
+
+Note, that in case of the last naming scheme example in the `table above <common_naming_scheme_violations_>`_, when you rename the binary RPM from ``<srcname>`` to ``python2-<srcname>``, the `%python_provide`_ macro will not provide the old name ``<srcname>``. To keep the upgrade path clean you will have to provide it and obsolete the old verision manually. You may place the tags right after the `%python_provide`_ macro:
+
+.. code-block:: spec
+   :emphasize-lines: 2,3
+
+   %{?python_provide:%python_provide python2-%{srcname}}
+   Provides:   <srcname> = %{version}-%{release}
+   Obsoletes:  <srcname> <= %{version}-%{release}
```

Comment:
Changed in  	c53d6d2. View [here](https://irushchyshyn.github.io/python-rpm-porting/naming-scheme.html#add-a-package-section-for-the-python-2-subpackage). What do you think?